source/_views/article.html

```diff
@@ -5,11 +5,11 @@
           <div class="col-md-9">
             <header>
               <h1 class="pio-docs-title">{{ page.title }}</h1>
-              <!-- <script type="text/javascript">
-            #  breadcrumbs()
-
-            </script> -->
-
+              <script type="text/javascript">
+                <!--
+                breadcrumbs()
+                //-->
```

Comment:
@bensheldon I thought that was odd too, but it's straight from the source. @rachelwhitton can you test without the comments?